### DISCUSSION TITLE
Simulation smoother implemented in pymc-experimental?

### DISCUSSION THREAD

**Discussion by rklees** [pymc-devs/pymc-extras#339] [Q&A]
I wonder if someone knows whether pymc-experimental has implemented a simulation smoother? If so, where can I find more information about its use?

**Comment by ricardoV94** [comment]
I don't know what that is, and I'm sure there is nothing like it in the library. Do you have an easy to digest reference?

**Comment by rklees** [comment]
The only source I know is statsmodels. However, it provides the simulation smoother outputs as numpy arrays, not as inference data object. The equations are in the book of Durbin & Koopmans 2012.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jessegrabowski** [comment]
No, we only have the RTS smoother. Contributions are welcome if that's something important to you :)
